support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/support/oauth/web/response/accesstoken/ext/AccessTokenAuthorizationCodeGrantRequestExtractor.java

```diff
@@ -58,6 +59,11 @@ public AccessTokenRequestDataHolder extract(final HttpServletRequest request, fi
         if (token == null || token.isExpired()) {
             throw new InvalidTicketException(getOAuthParameter(request));
         }
+        val tgtId = token.getTicketGrantingTicket().getId();
+        val tgt = getOAuthConfigurationContext().getTicketRegistry().getTicket(tgtId, TicketGrantingTicket.class);
```

Comment:
This should be using `CentralAuthenticationService` instead of ticket registry. This breaks with registries that require transactional support, and also throwing the exception is not necessary since `CentralAuthenticationService` already does this.